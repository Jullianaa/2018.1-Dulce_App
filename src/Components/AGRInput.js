import React from 'react';
import {TextInput} from 'react-native';
import {Container, Content, Text, Input, Item, Form, Label} from 'native-base';

const styles = {
  textInput: {
    height: 40,
    borderBottomWidth: 1,
    borderBottomColor: 'grey',
    marginBottom: 10,
    width: '100%',
    alignSelf: 'center',
    marginTop: 30
  },
  container: {
    paddingHorizontal: 30
  }
};

class AGRInput extends React.Component {
  render() {
    return (
      <Form>
        <Item floatingLabel>
          <Label>{this.props.nameLabel}</Label>
          <Input style={styles.textInput} underlineColorAndroid='transparent'
            {...this.props} />
        </Item>
      </Form>
    );
  }
}

export default AGRInput;
