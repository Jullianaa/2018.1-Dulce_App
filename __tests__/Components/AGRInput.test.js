import React from 'react';
import renderer from 'react-test-renderer';
import AGRInput from '../../src/Components/AGRInput';
require('bezier');

it('renders correctly', () => {
  const tree = renderer
  .create(<AGRInput />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});
