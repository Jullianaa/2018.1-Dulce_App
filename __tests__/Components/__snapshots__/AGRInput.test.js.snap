// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={Object {}}
  virtual={undefined}
>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "transparent",
        "borderBottomWidth": 1,
        "borderColor": "#D9D5DC",
        "borderLeftWidth": 0,
        "borderRightWidth": 0,
        "borderTopWidth": 0,
        "flexDirection": "row",
        "marginLeft": 15,
        "marginTop": 15,
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <View
      collapsable={undefined}
      style={
        Object {
          "left": 0,
          "opacity": 1,
          "paddingBottom": undefined,
          "paddingTop": undefined,
          "position": "absolute",
          "right": 0,
          "top": 18,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#575757",
            "fontSize": 17,
            "left": 0,
            "paddingRight": 5,
            "top": 6,
          }
        }
        virtual={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "fontSize": 17,
            }
          }
          virtual={undefined}
        />
      </Text>
    </View>
    <TextInput
      allowFontScaling={true}
      editable={true}
      onBlur={[Function]}
      onChangeText={[Function]}
      onFocus={[Function]}
      placeholderTextColor="#575757"
      style={
        Array [
          Object {
            "color": "#000",
            "flex": 1,
            "fontSize": 17,
            "height": 50,
            "lineHeight": 24,
            "marginLeft": null,
            "paddingLeft": null,
            "paddingRight": 5,
            "top": 10,
          },
          Object {
            "alignSelf": "center",
            "borderBottomColor": "grey",
            "borderBottomWidth": 1,
            "height": 40,
            "marginBottom": 10,
            "marginTop": 30,
            "width": "100%",
          },
        ]
      }
      underlineColorAndroid="transparent"
      value=""
      virtual={undefined}
    />
  </View>
</View>
`;
